e2e/app/page/admin/user-profile-admin-page.ts

```diff
@@ -46,11 +46,11 @@ export default class UserProfileAdminPage extends BaseAdminPage {
   }
 
   async getDataAccessTiers(): Promise<string> {
-    return this.getStaticText('Data Access Tiers');
+    return this.getStaticText('Data access tiers');
```

Comment:
This test broke after #6402 so let's update the case of the texts

Reply:
Ohh shoot i did not know it was that tight! Thanks!